integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Domain.java

```diff
@@ -684,4 +698,33 @@ private static boolean executeCommandOnPod(V1Pod pod,
 
     return true;
   }
+
+  /**
+   * Copy a file from Kubernetes pod to local filesystem.
+   * @param namespace namespace of the pod
+   * @param pod name of the pod where the file is copied from
+   * @param container name of the container
+   * @param srcPath source file location on the pod
+   * @param destPath destination file location in local filesystem
+   * @return true if no exception thrown, false otherwise
+   */
+  private static boolean copyFileFromPod(String namespace,
+                                         String pod,
+                                         String container,
+                                         String srcPath,
+                                         Path destPath) {
+    try {
+      getLogger().info("Copy file {0} from pod {1} in namespace {2} to {3}", srcPath, pod, namespace, destPath);
+      Kubernetes.copyFileFromPod(namespace, pod, container, srcPath, destPath);
```

Comment:
Even though copyFileFromPod() was moved to FileUtils, you just going to call it directly in Kubernetes or shouldn't this be changed to use FileUtils?

Reply:
in Domain.java, all other methods are directly called from Kubernetes.java. That's why I did not change to use FileUtils.java

Reply:
Still doesn't make sense to me to even have it in FileUtils if we're not going to call through it.  copyFileToPod() is only called through FileUtils when I do a search so makes sense to me to do the same.  My .02